AMW_rest/src/main/java/ch/mobi/itc/mobiliar/rest/dtos/ConfigurationDTO.java

```diff
@@ -0,0 +1,19 @@
+package ch.mobi.itc.mobiliar.rest.dtos;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import lombok.Data;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@Data
+@XmlRootElement(name = "configuration")
+@XmlAccessorType(XmlAccessType.FIELD)
+@JsonAutoDetect()
+public class ConfigurationDTO {
```

Comment:
@RetGal is a separte dto realy needed, can't you annotation and return ApplicationConfigurationInfo, ConfigurationKeyValuePair and ConfigKey directly?
To obfuscate you could add a method to the existing service (like getObfuscated), there should already be a method that does this in the JSF module.

Reply:
I'm trying to figure out to get the properies of ConfigKey correctly into JSON/XML.
Obfuscation is done in the JSF-view directly, not in a methos:
`<h:outputText value="***************" rendered="${configItem.key.secretValue}" />`